software/index.html

```diff
@@ -36,96 +37,139 @@
-                        We introduce concepts and projects that are useful regardless of major: in the 21st century, having basic programming knowledge is vital no matter your hobbies or jobs you pursue.
-                    </p>
-                    <p class="fs-5 text-start">
-                        Sponsored by the Purdue College of Engineering, we are able to offer an official Purdue College of Engineering certification for everyone who attends!
+             <!-- Basic Info -->
+            <div class="my-3 p-5 container-lg fade-in-left">
+                <div class="d-flex flex-column">
+                    <h2 class="fs-1 text-light">Software Development <strong class="purdue-text">Simplified</strong>.</h2>
+                    <br/>
+                    <p class="fs-4 text-start text-light para-width">
+                         Software Saturdays is a free 8-week course, sponsored by Purdue IEEE and the College of Engineering, covering the basics of a software development framework, rotating every semester. This program is for any Purdue student who is seeking to learn a new software development framework, spruce up their resume, or is just interested in software. Additionally, everyone can earn a <strong class="purdue-text">College of Engineering</strong> certification upon completion. These look great on LinkedIn pages and resumes alike.
                     </p>
                 </div>
             </div>
 
-            <div class="my-2 p-5 container-lg">
-                <div class="row flex-wrap-reverse">
-                    <div class="col-md-10">
-                        <p class="fs-5 text-start">
-                            This semester, we are teaching Web Application Development through React.js, an ever-popular JavaScript framework. Even if you have no prior experience with React or web dev, our
-                            program will start from the basics - including HTML and CSS.
+             <!-- Semester Info -->
+            <div class="my-2 p-5 container-fluid text-light fade-in-right">
+                <div class="d-flex flex-row justify-content-end">
+                    <div>
+                        <h2 class="fs-1 text-light text-end">Spring 2023: <strong class="react-text">React</strong>.</h2>
+                        <br/>
+                        <p class="fs-4 text-start para-width">
+                            Join us to learn ReactJS in a series of 7 in-depth lessons. Covering the fundamentals of HTML, CSS, JS, and of course, <strong class="react-text">React</strong>. Put your skills to the test while building a full-stack web application using the <strong class="poke-text">Poke</strong>API and <strong class="firebase-text">Firebase</strong>. We currently have a beginner and intermediate track to support students at all points in their software development journey so even if your aren't too confident in your skills, still come check us out!
```

Comment:
This section is begging for an image on the left, it looks blank without it.

Reply:
Added image and had it get removed for mobile since it just doesn't fit well on a mobile page